modules/compiler/spirv-ll/test/spvasm/op_spec_constant_composite_uint_struct.spvasm

```diff
@@ -39,12 +39,11 @@
        %void = OpTypeVoid
           %3 = OpTypeFunction %void
         %uint = OpTypeInt 32 0
-   %idx_type = OpTypeInt 32 0
         %Foo = OpTypeStruct %uint
  %test_block = OpTypeStruct %Foo
 %_ptr_Uniform_test_block = OpTypePointer Uniform %test_block
           %_ = OpVariable %_ptr_Uniform_test_block Uniform
-         %11 = OpConstant %idx_type 0
+         %11 = OpConstant %uint 0
```

Comment:
Any reason for this change?

Reply:
`%idx_type` is just `%uint` (they're both `OpTypeInt 32 0`) and it's invalid to define any type more than once.